core/src/main/java/jenkins/util/io/PathRemover.java

```diff
@@ -100,7 +100,7 @@ public void forceRemoveRecursive(@NonNull Path path) throws IOException {
     @Restricted(NoExternalUse.class)
     @FunctionalInterface
     public interface PathChecker {
-        void check(@NonNull Path path) throws SecurityException;
+        void check(@NonNull Path path);
```

Comment:
[Effective Java 3rd Edition §74](https://books.google.com/books?id=BIpDDwAAQBAJ) says:

> Use the Javadoc `@throws` tag to document each exception that a method can throw, but do not use the `throws` keyword on unchecked exceptions.

This PR removes the `throws` keyword on this unchecked exception (good), but it does not add the Javadoc `@throws` tag (bad).